src/lib/tabs/tab-body.scss

```diff
@@ -1,6 +1,7 @@
 .mat-tab-body-content {
   height: 100%;
   overflow: auto;
+  overflow: unset;
```

Comment:
* Won't this undo the `overflow: auto` directly above this?
* This would definitely need a detailed explanation of what/why.

Reply:
1) Yes, LOL. I forgot to remove that. It doesn't seem that rule is really applied given the parent container is the element actually getting the overflow properties. @andrewseguin - any ideas what that was for?
2) Added.

Reply:
@amcdnl Hard to say to be honest, we played a lot with the CSS on these tabs to get them to scroll right. Does each browser look alright without it? Check both dynamicHeight and non-dynamicHeight elements